script.js

```diff
@@ -0,0 +1,329 @@
+
+          keyElement.addEventListener('click', () => {
+            this.toggleCapsLock();
+          });
+
+          break;
+
+        case 'Enter':
+          keyElement.classList.add('keyboard__key_wide');
+          keyElement.innerHTML = '<span>Enter</span>';
+
+          keyElement.addEventListener('click', () => {
+            this.properties.value = document.querySelector('.use-keyboard-input').value;
+            this.properties.value += '\n';
+            document.querySelector('.use-keyboard-input').value = this.properties.value;
+          });
+
+          break;
+
+        case 'Shift':
+          keyElement.classList.add('keyboard__key_wide');
+          keyElement.innerHTML = '<span>Shift</span>';
+
+          break;
+
+        case 'Control':
+          keyElement.classList.add('keyboard__key_wide');
+          keyElement.innerHTML = '<span>Control</span>';
+
+          break;
+
+        case 'Alt':
+          keyElement.classList.add('keyboard__key_wide');
+          keyElement.innerHTML = '<span>Alt</span>';
+
+          break;
+
+        case 'Space':
+          keyElement.classList.add('keyboard__key_extra-wide');
+          keyElement.innerHTML = '<span> </span>';
+
+          keyElement.addEventListener('click', () => {
+            this.properties.value = document.querySelector('.use-keyboard-input').value;
+            this.properties.value += ' ';
+            document.querySelector('.use-keyboard-input').value = this.properties.value;
+          });
+
+          break;
+
+        default: {
+          let lenguageLetter;
+          if (!this.properties.isRrussian) {
+            // eslint-disable-next-line prefer-destructuring
+            lenguageLetter = key[0];
+          } else {
+            // eslint-disable-next-line prefer-destructuring
+            lenguageLetter = key[1];
+          }
+
+          keyElement.textContent = lenguageLetter.toLowerCase();
+
+          keyElement.addEventListener('click', () => {
+            if (this.properties.capsLock) {
+              this.properties.value = document.querySelector('.use-keyboard-input').value;
+              this.properties.value += lenguageLetter.toUpperCase();
+            } else {
+              this.properties.value = document.querySelector('.use-keyboard-input').value;
+              this.properties.value += lenguageLetter.toLowerCase();
+            }
+
+            document.querySelector('.use-keyboard-input').value = this.properties.value;
+          });
+
+          break;
+        }
+      }
+
+      fragment.appendChild(keyElement);
+
+      if (insertLineBreak) {
+        // add br after some keys
+        fragment.appendChild(document.createElement('br'));
+      }
+    });
+
+    return fragment;
+  }
+
+  toggleCapsLock() {
+    this.properties.capsLock = !this.properties.capsLock;
+
+    this.elements.keys.forEach((key) => {
+      const myKey = key;
+      if (myKey.childElementCount === 0) {
+        if (this.properties.capsLock) {
```

Comment:
avoid structure with difficult enclosure, they are very comprehensive to understand and debug